case_studies/weak_lensing/lensing_convnet.py

```diff
@@ -22,15 +22,13 @@ def __init__(self, n_bands, ch_per_band, num_features, tile_slen):
         module_list = []
 
         for _ in range(self.n_downsample):
-            module_list.append(
-                ConvBlock(nch_hidden, 2 * nch_hidden, kernel_size=5, stride=2, padding=2)
-            )
+            module_list.append(ConvBlock(nch_hidden, 2 * nch_hidden, kernel_size=5, stride=2))
             nch_hidden *= 2
 
         module_list.extend(
             [
-                ConvBlock(nch_hidden, 64, kernel_size=5, padding=2),
-                nn.Sequential(*[ConvBlock(64, 64, kernel_size=5, padding=2) for _ in range(1)]),
+                ConvBlock(nch_hidden, 64, kernel_size=5),
+                nn.Sequential(*[ConvBlock(64, 64, kernel_size=5) for _ in range(1)]),
```

Comment:
@shreyasc30 @timwhite0 The appropriate padding gets determined automatically now by `ConvBlock`, so `padding` is no longer an available argument.